src/components/share-overlay/share-overlay.js

```diff
@@ -607,7 +610,8 @@ class ShareOverlay extends Component {
    * @memberof ShareOverlay
    */
   _handleClipStartTimeChange = (value: string): void => {
-    this.setState({clipStartTimeValue: this._convertTimeValue(value)});
+    const newTime = Math.floor(this._convertTimeValue(value) / 2) * 2;
```

Comment:
can value be 0 ?

Reply:
Yes, it can be zero, and if it is zero - as a result we still get 0